query-primitives/src/main/scala/io/shiftleft/queryprimitives/steps/types/structure/Method.scala

```diff
@@ -300,6 +311,11 @@ class Method[Labels <: HList](override val raw: GremlinScala.Aux[nodes.Method, L
   def block: Block[Labels] =
     new Block[Labels](raw.out(EdgeTypes.AST).hasLabel(NodeTypes.BLOCK).cast[nodes.Block])
 
+  /**
+    * Traverse to method body (alias for `block`)
+    * */
+  def body: Block[Labels] = isBlock
```

Comment:
This does not seem to work because `isBlock` is just a filter. Did you intend to use `block`?

Reply:
Fixed